web/src/routes/admin/(resources)/students/+page.svelte

```diff
@@ -0,0 +1,42 @@
+<script lang="ts">
+  import DataTable, { type DataItem } from '$lib/elements/ui/dataTable/data-table.svelte';
+  import { deleteStudent, getStudents, type Pageable, type PageStudent } from '$lib/sdk/fetch-client';
+  import { error } from '@sveltejs/kit';
+  import { onMount } from 'svelte';
+  import { writable, type Writable } from 'svelte/store';
+
+  let columnNames = ['Vorname', 'Nachname', 'Tags'];
+  let keys = ['id', 'firstName', 'lastName', 'tags'];
+  let tableData: Writable<DataItem[]> = writable();
+  let totalElements: Writable<number> = writable(0);
+
+  onMount(async () => loadPage(0, '', ''));
+
+  async function loadPage(index: number, sortString: string, filter: string) {
+    let pageable: Pageable = { page: index, size: 10, sort: [sortString] };
+    try {
+      const result: PageStudent = await getStudents(pageable, {
+        firstName: filter,
+        lastName: filter,
+      });
+      totalElements.set(result.totalElements as number);
+      tableData.set(result.content as unknown as DataItem[]);
+    } catch {
+      error(404, { message: 'Could not fetch page' });
+    }
+  }
+
+  async function deleteEntry(id: string) {
+    try {
+      await deleteStudent(id);
+    } catch {
+      error(400, { message: `student with id ${id} could not be found` });
```

Comment:
In this case you could argue that it's a 404

Reply:
Changed the message to better fit the 400 error code, to resemble the other errors with code 400.

Reply:
That's generally an interesting debate. You *can* argue that — since `id` is a path variable — accessing `/students/:id` results in a 404 page not found, as that path doesn't exist if the id doesn't exist. However, since we're not talking about static paths here, some people also say that a 404 is inappropriate.